client/widgets/Images.cpp

```diff
@@ -454,7 +455,15 @@ void CShowableAnim::blitImage(size_t frame, size_t group, SDL_Surface *to)
 	Rect src( xOffset, yOffset, pos.w, pos.h);
 	auto img = anim->getImage(frame, group);
 	if(img)
-		img->draw(to, pos.x, pos.y, &src, alpha);
+	{
+		if(alpha != UINT8_MAX)
```

Comment:
This one is a bit suspicious - it is possible for image to remain slightly transparent?
When alpha value reaches 255 your image will still have its alpha value from previous loop.

I think you need to either remove this check or reset image alpha back to 255 after draw()

Reply:
Indeed